viewing_party/party.py

```diff
@@ -0,0 +1,222 @@
+# Wave 1
+# -------------------
+
+# first 4 tests
+def create_movie(title, genre, rating):
+    new_movie = {}
+    if title and genre and rating:
+        new_movie["title"] = title 
+        new_movie["genre"] = genre
+        new_movie["rating"] = rating
+        return new_movie
```

Comment:
It's cleaner (and less typing) if you build the dictionary all at once:

```suggestion
    if title and genre and rating:
        new_movie = {
            "title": title,
            "genre": genre,
            "rating": rating
        }
        return new_movie
```